mathematica.html

```diff
@@ -0,0 +1,11747 @@
+
```

Comment:
I don't think we can include this content here according to the posted policy at https://docs.github.com/en/free-pro-team@latest/github/site-policy/github-terms-of-service#f-copyright-infringement-and-dmca-policy

> You may not duplicate, copy, or reuse any portion of the HTML/CSS, Javascript, or visual design elements or concepts without express written permission from GitHub.

Reply:
Good point. :-) Didn't even think of this.  Can you think of another site that perhaps wouldn't care if we borrowed an HTML sample without just making one up?  I'm not sure GitHub would come after us for this, but if you want to err on the side of caution I also understand that. :)

Reply:
If we were to add a "source code" sample test would any large OSS project with a permissive license work? (ie Vue, React, etc?)

Reply:
I'm a Svelte maintainer and would be happy if you want to use svelte.dev or kit.svelte.dev. Our sites are MIT licensed